src/components/MovieDetails/MovieDetails.js

```diff
@@ -20,23 +20,41 @@ class MovieDetails extends Component {
             .catch(() => this.setState({ error: 'Having trouble finding this movie right now...please try again.'} ));
     }
 
-    // formatOverviewLength = (overview) => {
-    //     if (overview.length && overview.length > 20) {
-    //         const shortOverview = overview.slice(0, 20) + '...';
-    //         console.log('Shorty', shortOverview)
-    //         return shortOverview;
-    //     } else {
-    //         console.log('OVERVIEW', overview)
-    //         return overview;
-    //     }
-    // }
+    formatOverview = (overview) => {
+        if (overview.length > 185) {
+            const shortOverview = overview.slice(0, 185) + '...';
+            return shortOverview;
+        } else {
+            return overview;
+        }
+    }
+
+    formatGenres = (genres) => {
+        const allGenres = genres.join(' | ')
+        return allGenres 
+      }
+
+    formatRating = (rating) => {
+        const formattedRating = rating.toFixed(0)
+        return formattedRating
+    }
+    
+    formatReleaseDate = (date) => {
+        const yearReleased = date.split("-")[0]
+        return yearReleased
+    }
    
     render() {
         
-        if (this.state.movieInfo.id) {
```

Comment:
This conditional can actually be removed now and we can destructure this.state.movieInfo and you won't see any errors now when the MovieDetails component renders